lib/elixir/test/elixir/enum_test.exs

```diff
@@ -151,10 +151,17 @@ defmodule EnumTest do
 
   test "fetch/2" do
     assert Enum.fetch([2, 4, 6], 0) == {:ok, 2}
+    assert Enum.fetch([2, 4, 6], -1) == {:ok, 6}
     assert Enum.fetch([2, 4, 6], 2) == {:ok, 6}
     assert Enum.fetch([2, 4, 6], 4) == :error
     assert Enum.fetch([2, 4, 6], -2) == {:ok, 4}
     assert Enum.fetch([2, 4, 6], -4) == :error
+
+    # optimizations
```

Comment:
good catch.
I would still leave them to tests lists with only one element.
Are you ok with that?
I also will add a test for an empty list